cmd/start.go

```diff
@@ -24,6 +24,7 @@ const (
 	bearerTokenFlagName    = "bearer-token"
 	corsFlagName           = "cors-origin"
 	syncProviderFlagName   = "sync-provider"
+	prettyLogFlagName      = "log-format"
```

Comment:
```suggestion
	logFormatFlagName      = "log-format"
```

Please also update all the references to this variable.

Reply:
done